CHANGELOG.md

```diff
@@ -8,14 +8,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-### Added
+## Added
 
 - ERC1155 component and preset (#896)
+- Mixin implementations in components (#863)
+  - `AccessControlComponent::AccessControlMixinImpl`
+  - `OwnableComponent::OwnableMixinImpl`
+  - `OwnableComponent::OwnableTwoStepMixinImpl`
+  - `AccountComponent::AccountMixinImpl`
+  - `EthAccountComponent::EthAccountMixinImpl`
+  - `ERC20Component::ERC20MixinImpl`
+  - `ERC721Component::ERC721MixinImpl`
+  - `ERC721Receiver::ERC721ReceiverMixinImpl`
```

Comment:
maybe this level of detail is not that important for this addition, at most mentioning the modules (Accounts, ERC20, ERC721, etc) but maybe not even

Reply:
That's fair. I figured it's better to risk too much than too little. I'll whittle this down